src/index.ts

```diff
@@ -23,32 +40,10 @@ async function main() {
 		}
 	});
 
-	const q = async.queue<Store>(async (store: Store, cb) => {
-		setTimeout(async () => {
-			try {
-				Logger.debug(`↗ scraping initialized - ${store.name}`);
-				await lookup(browser, store);
-			} catch (error) {
-				// Ignoring errors; more than likely due to rate limits
-				Logger.error(error);
-			} finally {
-				cb();
-				q.push(store);
-			}
-		}, Config.browser.rateLimitTimeout);
```

Comment:
I would keep this queue logic and use the function `getSleepTime` you created.

Is there a reason why we would delete this queue logic?

Reply:
The queue can theoretically drain even with multiple stores, which would then close the browser.
I don't see the reason for that queue in the first place. It doesn't increase parallelism at all.
And it can never drain (because all functions loop infinitely).